srcpkgs/mlpack/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'mlpack'
+pkgname=mlpack
+version=3.2.2
+revision=1
+build_style=cmake
+hostmakedepends="txt2man"
+makedepends="armadillo-devel boost-devel"
+depends="armadillo boost"
+short_desc="Scalable C++ machine learning library"
+maintainer="toluschr <toluschr@protonmail.com>"
+license="BSD-3-Clause"
+homepage="https://mlpack.org/"
+distfiles="https://github.com/mlpack/mlpack/archive/${version}.tar.gz"
+checksum=9dcdcbd6c1e42a7362ee64adc2afb96241f499590b795a7713c39bb6e83c610a
+
+post_install() {
+	vlicense LICENSE.txt
+}
+
+libmlpack_package() {
+	short_desc+=" - shared libraries"
+
+	pkg_install() {
+		vmove "usr/lib/*.so*"
+	}
+}
+
+libmlpack-devel_package() {
+	short_desc+=" - development files"
+	depends="${makedepends} libmlpack>=${pkgname}_${version}"
```

Comment:
should be rather `libmlpack>=${version}_${revision}`

Reply:
oh I completely messed up